jerry-core/parser/js/js-parser.c

```diff
@@ -2498,7 +2537,18 @@ parser_parse_function (parser_context_t *context_p, /**< context */
 #ifdef PARSER_DUMP_BYTE_CODE
   if (context_p->is_show_opcodes)
   {
+#ifndef CONFIG_DISABLE_ES2015_CLASS
+    if (context_p->status_flags & PARSER_CLASS_CONSTRUCTOR)
+    {
+      JERRY_DEBUG_MSG ("\n--- Class constructor parsing start ---\n\n");
+    }
+    else
+    {
+      JERRY_DEBUG_MSG ("\n--- Function parsing start ---\n\n");
```

Comment:
Does it look better with a `?:` or the line is too long?

Reply:
It looks better but the lines are too long :/